ably/presence.go

```diff
@@ -1,27 +1,22 @@
-package rest
-
-import (
-	"github.com/ably/ably-go/config"
-	"github.com/ably/ably-go/proto"
-)
+package ably
 
 type Presence struct {
-	client  *Client
-	channel *Channel
+	client  *RestClient
```

Comment:
You do have Presence on the realtime side too, so I would probably take the occasion to rename presence `RestPresence` and add a getter on `RestChannel` for `Presence`.